mathics_django/web/media/js/mathics.js

```diff
@@ -1,829 +1,881 @@
 // TODO: getSelection function
+// -*- mode: js; js-indent-level: 4; -*-
```

Comment:
`.editorconfig`  in the root directory says indent_style is space and indent-level is 4. 
For some reason Emacs is not picking up editorconfig so this makes it explicit here.

Reply:
Sorry for all of the space related changes below which turn tabs into spaces. I will mark the significant changes though.